node-tests/blueprints/mixin-test.js

```diff
@@ -197,4 +290,53 @@ describe('Blueprint: mixin', function() {
       });
     });
   });
+
+  describe('in in-repo-addon - module unification', function() {
```

Comment:
I haven't yet landed https://github.com/ember-cli/ember-cli/pull/7783 so you can remove this test for now, it's not currently possible to test the desired behaviour of MU `in-repo-addon` blueprint output. I'll be sure to follow up and add these tests later